boards/arm/arduino_nano_33_ble/doc/index.rst

```diff
@@ -103,6 +103,12 @@ For example
 
     west flash --bossac=$HOME/.arduino15/packages/arduino/tools/bossac/1.9.1-arduino2/bossac
 
+On Windows you also need to specify the COM PORT:
+
+.. code-block:: bash
+
+    west flash --bossac=%USERPROFILE%\AppData\Local\Arduino15\packages\arduino\tools\bossac\1.9.1-arduino2\bossac.exe --bossac-port="COMx"
```

Comment:
-p instead --bossac-port doesn't work?

Reply:
-p with bossac is not working on windows:
C:\work\zephyr-fork\zephyr\samples\basic\blinky>west flash --bossac="%USERPROFILE%\AppData\Local\Arduino15\packages\arduino\tools\bossac\1.9.1-arduino2\bossac.exe" -p "COM32"
-- west flash: rebuilding
ninja: no work to do.
-- west flash: using runner bossac
FATAL ERROR: runner bossac received unknown arguments: ['-p', 'COM32']

C:\work\zephyr-fork\zephyr\samples\basic\blinky>west flash --bossac="%USERPROFILE%\AppData\Local\Arduino15\packages\arduino\tools\bossac\1.9.1-arduino2\bossac.exe" -p COM32
-- west flash: rebuilding
ninja: no work to do.
-- west flash: using runner bossac
FATAL ERROR: runner bossac received unknown arguments: ['-p', 'COM32']